src/commons/stores/user-store.js

```diff
@@ -0,0 +1,69 @@
+(function() {
+    "use strict";
+
+    angular
+        .module("TendrlModule")
+        .service("userStore", userStore);
+
+    /*@ngInject*/
+    function userStore($state, $q, userFactory) {
+        var store = this;
+
+        store.getUserList = function() {
+            var deffered;
+
+            deffered = $q.defer();
+            userFactory.getUserList()
+                .then(function(data) {
+                    deffered.resolve(data);
+                });
+
+            return deffered.promise;
+        };
+
+        store.generateName = function(name) {
+            var userName = [],
+                temp1 = [];
+
+            temp1 = name.split(' ');
+            userName.firstName = temp1[0];
+            userName.lastName = temp1[1];
+            return userName;
+        };
+
+
+        store.setupUserListData = function(data) {
+            var i,
+                j,
+                length = data.length,
+                userList = [],
+                index,
+                user;
+
+            for (i = 0; i < length; i++) {
+                user = {};
+                user.username = data[i].username;
+                user.firstName = store.generateName(data[i].name).firstName;
+                user.lastName = store.generateName(data[i].name).lastName;
+                if (data[i].status === 'true') {
+                    user.status = 'enabled';
```

Comment:
@a2batic We follow double quotes across the UI app. Please change the single quote to double quotes to make it consistent across the app.